reports/report_builder.py

```diff
@@ -3029,7 +3029,24 @@ def ws_100(self, ws):
         return
 
     def ws_101(self, ws):
-        return
+        counts_list = []
+        for _, model in sheet_models.items():
+            counts = Counter()
+            sex = model.journalist_field_name() + '__sex'
+
+            rows = model.objects \
+                .values('covid19', 'topic', sex, 'country') \
+                .filter(**{model.journalist_field_name() + '__sex__in': self.male_female_ids}) \
```

Comment:
What effect does this have i.e. is it possible to have sex not defined in `male_female_ids`?

Reply:
From what I understand at other places where this was used, I think there are stories that don't have `sex` defined. I'm not sure though but that was what I inferred from context. @kilemensi